README.md

```diff
@@ -2,3 +2,13 @@
 
 # sentiment-analysis
 Simple show case of consuming twitter api and performing sentiment analysis over tweets
+
+### How to contribute
+You need the following dependencies installed:
+- [Gradle](https://gradle.org/)
+- [ElasticSearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/_installation.html)
+- [Kibana](https://www.elastic.co/downloads/kibana)
+
+### How to run it
+After installing `Elasticsearch` just go to the bin directory and fire up `elasticsearch`
+after that you need to run `gradle run` 
```

Comment:
also for windows users run bin/elasticsearch

Reply:
Sure, will do it.